content/applications/websites/ecommerce/managing_products/products.rst

```diff
@@ -0,0 +1,265 @@
+     be enabled under :menuselection:`Website --> Configuration --> Settings --> Shop - Products`.
+     You can learn more about variants
+     :doc:`here <../../../sales/sales/products_prices/products/variants>`.
+
+Layout
+------
+
+Within the same :guilabel:`Customize` tab as the :ref:`functions <ecommerce-functions>`, the layout
+configuration can be changed according to your needs.
+
+- :guilabel:`Images Width`: changes the width of the image product displayed on the page;
+- :guilabel:`Layout`: the :guilabel:`Carousel` layout displays a large, main image with smaller ones
+  underneath; whereas the :guilabel:`Grid` displays four images in a square layout (see pictures
+  below);
+- :guilabel:`Image Zoom`: if the zoom occurs on :guilabel:`Pop-up on Click`, when hovering over the
+  image (:guilabel:`Magnifier on hover`), on :guilabel:`Both`, or :guilabel:`None`;
+- :guilabel:`Thumbnails`: if you wish to align the thumbnails **vertically** (:guilabel:`Left`), or
+  **horizontally** (:guilabel:`Right`);
+- :guilabel:`Main Image`: click :guilabel:`Replace` to change the product's main image;
+- :guilabel:`Extra Images`: click :guilabel:`Add` or :guilabel:`Remove all` to add or remove product
+  images. You can also add images and videos via **URL**.
+
+.. note::
+   Images must either be PNG or JPG, and at least 1042px in height or width. To trigger the zoom,
+   the image has to be bigger than 1024x1024.
+
+.. image:: products/products-layout.png
+   :align: center
+   :alt: Product images layout
+
+Add content
+-----------
+
+You can use **building blocks** (:menuselection:`Edit --> Blocks`) to add content to your product
+page. These blocks can be used to add extra text and picture galleries, features such as
+:guilabel:`Call to Actions`, :guilabel:`Comparisons`, etc. Depending on *where* you drop the
+**building block**, it may be available either on the product page *only*, or on the *whole*
+website.
+
+.. image:: products/products-blocks.png
+   :align: center
+   :alt: Building blocks on product page
+
+Download link
+-------------
+
+If you wish to add a downloadable file (ex.: user's manual, notice of use, etc.) on the product
+page, you can do so by adding a :guilabel:`Text` block from :menuselection:`Edit --> Blocks` on the
+page. Once placed, click within the :guilabel:`Text` block, and under the :guilabel:`Inline Text`
+section, select either :menuselection:`Insert Media --> Documents` or :guilabel:`Insert or edit
+link` and enter the URL in the :guilabel:`Your URL` field.
+
+.. note::
+   The difference with :ref:`digital files <ecommerce-digital-file>` is that digital files can only
+   be downloaded *after* checkout.
+
+.. image:: products/products-media.png
+   :align: center
+   :alt: Media and link buttons
+
+.. important::
+   The :ref:`developer mode <developer-mode>` is intended only for **advanced users** who want to
+   have access to advanced and optional features.
```

Comment:
If I recall correctly, Jon wants that users know the "developer mode" is not the intended working way and generally shouldn't even be in the doc. However, some options are just stuck behind the "dev mode."  For a user that reads the complete doc, it may get repetitive, but for someone specifically looking for a piece of information and landing on this page, I believe the "dev" note should be indicated.

I'm changing it from "Important" to "Note".

P.S.: Since it was linked to the studio "trick" that was removed, I removed it too.